### ISSUE TITLE
Question about how `queueIndex` in `XrGraphicsBindingVulkanKHR` is interpreted.

### ISSUE THREAD

**Issue by yshui** [KhronosGroup/OpenXR-Docs#168]
I assume after the XR runtime gets the queue, it would want to get the `VkQueue` by calling `vkGetDeviceQueue`?

But according to the spec, for `vkGetDeviceQueue`:

> VUID-vkGetDeviceQueue-flags-01841
> [VkDeviceQueueCreateInfo](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkDeviceQueueCreateInfo.html)::flags must have been set to zero when device was created

What if the application wants to use a queue created with non-zero flag? Doesn't seem to be a way to communicate the queue creation flag to the runtime.

**Comment by absolutelynothelix** [comment]
`vkGetDeviceQueue2`?

> To get queues that were created with a non-zero flags parameter use [vkGetDeviceQueue2](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceQueue2.html).

*from https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceQueue.html*

**Comment by yshui** [comment]
> `vkGetDeviceQueue2`?
> 
> > To get queues that were created with a non-zero flags parameter use [vkGetDeviceQueue2](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceQueue2.html).
> 
> _from [registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceQueue.html](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetDeviceQueue.html)_

Yeah, but even if the runtime still needs to know about the flags used to create the queue to use this function. And I don't think there is a way to give the runtime that info.
